tests/upgrades/test_errata.py

```diff
@@ -14,19 +14,17 @@
 
 :Upstream: No
 """
-import os
-
 from fabric.api import execute
 from nailgun import entities
 from robottelo.api.utils import call_entity_method_with_timeout
 from robottelo.constants import (
-    DISTRO_DEFAULT,
+    DISTRO_RHEL7,
```

Comment:
Below under tests, I saw that we are performing operations with rhel7 (e.g : https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_errata.py#L224 ) but while choosing the version we are choosing the default. This will fail when DEFAULT will switch to rhel8, hence to avoid that I stick to RHEL7.